pages/dkp/kaptain/1.3.0/uninstall/index.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: layout.pug
+navigationTitle: Uninstall
+title: Uninstall Kaptain
+menuWeight: 7
+excerpt: Uninstall Kaptain from your cluster
+beta: false
+enterprise: false
+---
+
+## Prerequisites
+
+-   Ensure you have a backup of your Kaptain installation if you are migrating to Kaptain 2.x.
```

Comment:
Do we need to mention specific instances, parts of Kaptain that should be backed up? Or is this general statement enough?